parser/metadecoders/decoder.go

```diff
@@ -242,9 +242,10 @@ func (d Decoder) unmarshalORG(data []byte, v any) error {
 		k = strings.ToLower(k)
 		if strings.HasSuffix(k, "[]") {
 			frontMatter[k[:len(k)-2]] = strings.Fields(v)
-		} else if k == "tags" || k == "categories" || k == "aliases" {
-			log.Printf("warn: Please use '#+%s[]:' notation, automatic conversion is deprecated.", k)
-			frontMatter[k] = strings.Fields(v)
+		} else if strings.HasSuffix(k, "[]") {
```

Comment:
You have a repeated condition here.

Reply:
Oh my, thanks for catching that. I've just fixed the commit.